MagnetoSoftwareTesting/src/test/java/MagnetoTest/MagnetoApplicationTest.java

```diff
@@ -0,0 +1,164 @@
+package MagnetoTest;
+
+import java.util.concurrent.TimeUnit;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+
+
+public class MagnetoApplicationTest {
+public String Url ="https://magento.softwaretestingboard.com";  
+//String driverpath ="C:\\Users\\rajashrees\\Desktop\\Seljars\\chromedriver_win32 (4)\\chromedriver.exe"; 
+//String driverName = "webdriver.chrome.driver";  
+public WebDriver driver; 
+
+@Parameters("browser")
+@BeforeTest  
+public void browser(String browser)  
```

Comment:
Please provide proper comments on each method for its usage.

Reply:
Add comments on each method.